build.sbt

```diff
@@ -12,7 +12,7 @@ licenses += ("Apache-2.0", url("https://opensource.org/licenses/Apache-2.0"))
 
 lazy val scalaV = "2.12.8"
 lazy val catsV = "1.6.0"
-lazy val parboiledV = "2.1.5"
+lazy val parboiledV = "2.1.6"
```

Comment:
Do we want to include this parboiled upgrade still? (I'm fine either way, just checking since it ended up being unrelated to the issue.)

Reply:
I'd say why not, always good to upgrade IMO as long as nothing breaks

Reply:
:+1: